genmc-sys/src/lib.rs

```diff
@@ -205,6 +258,7 @@ mod ffi {
         /**** Types shared between Miri/Rust and GenMC/C++: ****/
```

Comment:
```suggestion
        /**** Types shared between Miri/Rust and GenMC/C++:
            (This tells cxx that the enums defined above are already defined on the C++ side;
            it will emit assertions to ensure that the two definitions agree.) ****/
```